addons/loyalty/controllers/portal.py

```diff
@@ -85,6 +85,11 @@ def portal_my_loyalty_card_history(self, card_id, page=1, sortby='date', **kw):
 
     @route('/my/loyalty_card/<int:card_id>/values', type='jsonrpc', auth='user')
     def portal_get_card_history_values(self, card_id):
+        """
+        Retrieve card history values for portal card dialog.
+        :param card_id(str): The ID of the loyalty card.
+        :return(dict): A dictionary with card history values and trigger products.
```

Comment:
When website_sale is not installed, this function is stil called but no trigger products are returned since they are retirned on an override.
```suggestion
        :return(dict): A dictionary with card history values.
```